src/associates/README.md

```diff
@@ -0,0 +1,24 @@
+# associates
+
+__In Terminal:__
+
+To install necessary packages: 'npm install'
+
+To start webpack: 'npm run react-dev'
+
+To start server: 'npm start'
+
```

Comment:
Add line that if you are changing the styling for the associates widget, run 'npm run build' if that is correct.